r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	      : request-promise-native
+# Version	      : 1.0.7
+# Source repo     : https://github.com/request/request-promise-native
+# Tested on    	  : Ubuntu 18.04 (Docker)
+# Language        : Node
+# Travis-Check    : True
+# Script License  : Apache License, Version 2 or later
+# Maintainer	  : Sumit Dubey <sumit.dubey2@ibm.com> 
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+set -ex
+
+WORK_DIR=`pwd`
+
+PACKAGE_NAME=request-promise-native
+PACKAGE_VERSION=v1.0.7                 
```

Comment:
Can you make this PACKAGE_VERSION as configurable ?

Reply:
sure I will make this PACKAGE_VERSION as configurable